test/views/CheckYourAnswersViewSpec.scala

```diff
@@ -31,7 +31,7 @@ class CheckYourAnswersViewSpec extends ViewBehaviours {
 
```

Comment:
maybe a test to be added to assert the new sub heading stuff to ensure hierarchy. :)